tests/src/test/320+-noncdh-nondb/scala/org/apache/spark/sql/rapids/RapidsShuffleThreadedWriterSuite.scala

```diff
@@ -118,14 +118,23 @@ class BadSerializable(i: Int) extends Serializable {
   }
 }
 
+// Added so that we can mock a method that was created in Spark 3.3.0
+class TestIndexShuffleBlockResolver(
+    conf: SparkConf,
+    bm: BlockManager)
+      extends IndexShuffleBlockResolver(conf, bm) {
+  // implemented in Spark 3.3.0
+  def createTempFile(file: File): File = { null }
```

Comment:
`[ERROR] [Error] /home/peixinl/projects/spark-rapids/tests/src/test/320+-noncdh-nondb/scala/org/apache/spark/sql/rapids/RapidsShuffleThreadedWriterSuite.scala:127: overriding method createTempFile in class IndexShuffleBlockResolver of type (file: java.io.File)java.io.File;
 method createTempFile needs `override' modifier
[ERROR] one error found`

failed scala style check

Reply:
Thanks @pxLi. I am confused why this didn’t fail when I built locally… I’ll fix

Reply:
ah I was wrong, it was due to the trailing `-pl aggregator -am`, may need to remove them from the line

Reply:
let us make changes to build as a separate issue unless we can't unblock otherwise. 

I think there are multiple issues. Eg. there is no default value   `<maven.scalastyle.skip>false</maven.scalastyle.skip>` and thus mvn-verify-check workflow inadvertently skips scalastyle. Also it currently does not go beyond dist module and its transitive dependencies

Reply:
Agree. 
But if this one passed current CI, it would still fail the nightly build, so we may have a temp update for 330 to at least make sure 330 could have some sanity checks~

Reply:
Ok I opted for this solution for now: https://github.com/NVIDIA/spark-rapids/pull/6141/commits/41805234864b0aaddcb9d6de591bf653ed555d4d.

I am able to replicate the error @pxLi found and with this patch it goes away.

Reply:
and it's not scalastyle plugin it was a Scala testCompile for 330 complaining

Reply:
I have already updated above